Buildings/Fluid/Storage/Plant/StoragePlant.mo

```diff
@@ -0,0 +1,224 @@
+within Buildings.Fluid.Storage.Plant;
+model StoragePlant "Model of a storage plant with a chiller and a CHW tank"
+
+  replaceable package Medium =
+    Modelica.Media.Interfaces.PartialMedium "Medium package";
+
+  Buildings.Fluid.Sources.PropertySource_T chi(
+    redeclare final package Medium = Medium,
+    final use_T_in=true) "Chiller represented by an ideal temperature source"
+    annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=90,
+        origin={-50,-30})));
+  Modelica.Blocks.Sources.Constant TSet(final k=nom.T_CHWS_nominal)
+    "Constant CHW leaving temperature"
+    annotation (Placement(transformation(extent={{-100,-40},{-80,-20}})));
+  Buildings.Fluid.Movers.Preconfigured.FlowControlled_m_flow pumPri(
+    redeclare final package Medium = Medium,
+    final addPowerToMedium=false,
+    final m_flow_nominal=nom.mChi_flow_nominal,
+    final dp_nominal=chi2PreDro.dp_nominal) "Primary CHW pump"
+    annotation (Placement(transformation(extent={{-40,0},{-20,20}})));
+  Buildings.Fluid.FixedResistances.PressureDrop chi2PreDro(
+    redeclare final package Medium = Medium,
+    final m_flow_nominal=nom.mChi_flow_nominal,
+    dp_nominal=0.1*nom.dp_nominal)
+    "Pressure drop of the chiller branch in plant 2" annotation (Placement(
+        transformation(
+        extent={{10,-10},{-10,10}},
+        rotation=0,
+        origin={-30,-50})));
+  Buildings.Fluid.Storage.Plant.BaseClasses.TankBranch tanBra(
+    redeclare final package Medium = Medium,
+    final nom=nom,
+    VTan=0.8,
+    hTan=3,
+    dIns=0.3) "Tank branch, tank can be charged remotely" annotation (Placement(
+        transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=0,
+        origin={10,-10})));
+  Buildings.Fluid.Storage.Plant.BaseClasses.ReversibleConnection revCon(
+    redeclare final package Medium = Medium,
+    final nom=nom) "Reversible connection"
+    annotation (Placement(transformation(extent={{40,0},{60,20}})));
+  Buildings.Fluid.Storage.Plant.Controls.FlowControl floCon(
+    final mChi_flow_nominal=nom.mChi_flow_nominal,
+    final mTan_flow_nominal=nom.mTan_flow_nominal,
+    final use_outFil=true) "Control block for storage plant flows"
+    annotation (Placement(transformation(extent={{-60,40},{-40,60}})));
+  Buildings.Fluid.Storage.Plant.Controls.TankStatus tanSta(
+    TLow=nom.T_CHWS_nominal,
+    THig=nom.T_CHWR_nominal)
+    "Tank status"
+    annotation (Placement(transformation(extent={{40,-80},{60,-60}})));
+  parameter Buildings.Fluid.Storage.Plant.Data.NominalValues nom
+    "Nominal values for the storage plant"
+    annotation (Placement(transformation(extent={{60,60},{80,80}})));
+  Modelica.Blocks.Interfaces.BooleanInput chiIsOnl annotation (Placement(
```

Comment:
I suggest changing this input to be called ``chiEnaSta`` with comment "Chiller enable status, true if chiller is enabled", since it is more in line with convention.

Reply:
Revisions made as suggested in 4705f3bce0 and 0f2a49e038.